config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -176,3 +176,76 @@ periodics:
                 --ignore-destroy-errors \
                 --powervs-memory 32 \
                 --test=ginkgo -- --parallel 10 --flake-attempts 2 --test-package-bucket k8s-release-dev --test-package-dir ci --test-package-version $K8S_BUILD_VERSION --focus-regex='\[Conformance\]' --skip-regex='\[Disruptive\]|\[Serial\]'
+  - name: periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    interval: 3h
```

Comment:
Initially, we can run every 3hrs but the frequency can be reduced later.

Reply:
Noted.

Reply:
Also, @mkumatag should we think of including `kubetest2` installation in all-in-one image.
We see more jobs using them.

Reply:
then version of that bin becomes stale and already this image is really heavy, we also need to think about making it slim.!